packages/utils/lib/index.ts

```diff
@@ -95,8 +95,8 @@ export const localStorageOrDefault = <T>(
   key: string,
   _default: T,
   parse = false
-): T | string => {
-  const val: string | null = localStorage.getItem(key);
```

Comment:
This can be typed as `string | null` still. Local storage items are all strings.